ui/pages/create-account/import-account/import-account.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Box from '../../../components/ui/box';
+import NewAccountImportForm from '.';
+
+export default {
+  title: 'Pages/CreateAccount/ImportAccount',
```

Comment:
We can add component here
```suggestion
  title: 'Pages/CreateAccount/ImportAccount',
  component: NewAccountImportForm,
```